app/fiilee.py

```diff
@@ -0,0 +1,15 @@
+import json
+
+
+class Config:
+    def __init__(self, file_name: str="C:\\Users"
```

Comment:
The file path is hardcoded and uses backslashes, which are specific to Windows. Consider using a more flexible approach, such as using 'os.path.join' for cross-platform compatibility.